### PR TITLE
pacman-upgrade: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #5256

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

File: pages/linux/pacman-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --upgrade --help`
+
+- Install one or more packages from files:
+
+`sudo pacman --upgrade {{path/to/package1}} {{path/to/package2}}`
```
Comment (sbrl) [inline pages/linux/pacman-upgrade.md:12]: Do pacman packages have a file extension by convention? If so, it'd be nice to update these examples here to include it. Same goes for the other examples.

If not, just ignore this message.

File: pages/linux/pacman-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install a package without prompting:
+
+`sudo pacman --upgrade --noconfirm {{path/to/package}}`
+
+- Overwrite conflicting files during a package installation:
+
+`sudo pacman --upgrade --overwrite {{path/to/file}} {{path/to/package}}`
+
+- Install a package skipping the dependency version checks:
```
Comment (sbrl) [inline pages/linux/pacman-upgrade.md:22]: Perhaps this will read better?
```suggestion
- Install a package, skipping the dependency version checks:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some very minor comments below for you to review :slightly_smiling_face:

File: pages/linux/pacman-upgrade.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/pacman-upgrade.md:12]:
```suggestion
`sudo pacman --upgrade {{path/to/package1.pkg.tar.zst}} {{path/to/package2.pkg.tar.zst}}`
```

File: pages/linux/pacman-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`pacman --upgrade --help`
+
+- Install one or more packages from files:
+
+`sudo pacman --upgrade {{path/to/package1}} {{path/to/package2}}`
+
+- Install a package without prompting:
+
+`sudo pacman --upgrade --noconfirm {{path/to/package}}`
```
Comment (navarroaxel) [inline pages/linux/pacman-upgrade.md:16]:
```suggestion
`sudo pacman --upgrade --noconfirm {{path/to/package.pkg.tar.zst}}`
```

File: pages/linux/pacman-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Overwrite conflicting files during a package installation:
+
+`sudo pacman --upgrade --overwrite {{path/to/file}} {{path/to/package}}`
```
Comment (navarroaxel) [inline pages/linux/pacman-upgrade.md:20]:
```suggestion
`sudo pacman --upgrade --overwrite {{path/to/file}} {{path/to/package.pkg.tar.zst}}`
```

File: pages/linux/pacman-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install a package, skipping the dependency version checks:
+
+`sudo pacman --upgrade --nodeps {{path/to/package}}`
```
Comment (navarroaxel) [inline pages/linux/pacman-upgrade.md:24]:
```suggestion
`sudo pacman --upgrade --nodeps {{path/to/package.pkg.tar.zst}}`
```

File: pages/linux/pacman-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- List packages that would be affected (does not install any packages):
+
+`pacman --query --print {{path/to/package}}`
```
Comment (navarroaxel) [inline pages/linux/pacman-upgrade.md:28]:
```suggestion
`pacman --query --print {{path/to/package.pkg.tar.zst}}`
```

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**